frontend/src/container/Sidebar.jsx

```diff
@@ -35,16 +36,42 @@ import CustomToolboxContainer from '../component/CustomToolboxContainer.jsx'
 const qs = require('query-string')
 export const LOCK_TOGGLE_SIDEBAR_WHEN_OPENED_ON_MOBILE = 'lockToggleSidebarWhenOpenedOnMobile'
 
+export const SIDEBAR_STATE_LOCAL_STORAGE_KEY = {
+  FOLDED_SPACE_LIST: 'foldedSpaceList',
+  SHOW_SPACE_LIST: 'showSpaceList',
+  SHOW_USER_ITEMS: 'showUserItems'
+}
+export const buildSidebarStateLocalStorageKey = userId => `sidebarState/${userId}`
+export const getSidebarStateLocalStorage = userId => JSON.parse(
+  localStorage.getItem(buildSidebarStateLocalStorageKey(userId))
```

Comment:
Since `localStorage` is part of the `window` API, isn't it better to access it with `window.localStorage` for more clarity ?

Reply:
will fix

Reply:
fixed